src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -80,6 +81,81 @@ public function getLogs()
         return $this->data['logs'] ?? [];
     }
 
+    public function getProcessedLogs()
+    {
+        if (null !== $this->processedLogs) {
+            return $this->processedLogs;
+        }
+
+        $rawLogs = $this->getLogs();
+        if ([] === $rawLogs) {
+            return $this->processedLogs = $rawLogs;
+        }
+
+        $logs = [];
+        foreach ($this->getLogs()->getValue() as $rawLog) {
+            $rawLogData = $rawLog->getValue();
+
+            if ($rawLogData['priority']->getValue() > 300) {
+                $logType = 'error';
+            } elseif (isset($rawLogData['scream']) && false === $rawLogData['scream']->getValue()) {
+                $logType = 'deprecation';
+            } elseif (isset($rawLogData['scream']) && true === $rawLogData['scream']->getValue()) {
+                $logType = 'silenced';
+            } else {
+                $logType = 'regular';
+            }
+
+            $logs[] = [
+                'type' => $logType,
```

Comment:
should we add this is when processing logs in sanitizeLogs() already? then you'll have raw data, then we can spare out the array copying IIUC

the getLogs vs getProcessedLogs kinda confuses me :/